consensus/spos/bls/subroundBlock.go

```diff
@@ -589,3 +559,21 @@ func (sr *subroundBlock) getRoundInLastCommittedBlock() int64 {
 
 	return roundInLastCommittedBlock
 }
+
+func (sr *subroundBlock) updateLeaderPeerHonesty(node string) {
```

Comment:
Duplicate code with subroundEndRound. Suggestion to move this in Subround struct.